install-server.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+set -e
+
+echo -e "\n\nEverything is now getting setup. This process will take a few minutes...\n\n"
+
+sudo apt update && sudo apt install gpg jq -y && sudo apt clean
+
+#remove unattended upgrades
+sudo apt-get remove --purge unattended-upgrades || true
```

Comment:
i read this could get reinstalled as a dependency of another package and that the other approach is to disable it via config files. I don't feel too strongly about changing this over though as this is definitely easier to just get it removed/disabled

Reply:
Unattended upgrades likely won't be installed since this is ran on the default debian image. This was written to be ran on the debian cloud image, which had it. Most likely I may remove this line entirely since it isn't being ran on the debian cloud image anymore.